server/src/services/item.service.ts

```diff
@@ -1,13 +1,29 @@
 import Item from '../models/item.model';
 import ItemCategory from '../models/itemCategory.model';
-import SORT_TYPE from '../utils/SortType';
 import { Op, Transaction, WhereOptions } from 'sequelize';
 import ItemOption from '../models/itemOption.model';
 import ItemError, { ItemErrorType } from '../errors/item.error';
+import esClient from '../config/elasticsearch';
 import ItemTag from '../models/itemTag.model';
 import GetItemsRequest from '../requests/item/getItems.request';
 
 class ItemService {
+  async _searchToElasticSearch(keyword: string) {
+    const result = await esClient.search({
```

Comment:
```suggestion
    const result = await esClient.search<Item>({
```

이렇게 타입을 지정하면 아래에서 형변환을 진행하지 않아도 괜찮네요 👀

Reply:
오 그렇군요!